### ISSUE TITLE
Represent native contracts as interop packages

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#1691]
Anything can be invoked with `contract.Call` interop, but it's not very convenient for frequently used native contracts. Especially given the fact that we don't provide their hashes at the `interop` level (take a look at NeoFS `alphabet` contract for example). As more and more things move from syscalls to native contracts we should make it easier to use them. Most of other toolchains provide wrapper classes and we should probably make additional packages (one package --- one contract) with functions.

It's not really hard, but I'm not sure we can correctly make it without #941 (essentially some interop functions change to regular functions that use `contract.Call` inside (though it'd be nice to inline these calls)). It'd also be nice to autogenerate contract hashes for `interop` use and then check them in tests.

**Comment by fyrchik** [comment]
I think for this we can support some restricted form of inlining when function body consists of a single `return` or invocation and arguments are passed as they are (i.e. no function calls or operations). This should be both simple to implement (just keep track of map to "real arguments" vs all locals in general case) and enough to provide this helpers (which are all of form `return contract.Call(...)`.

However without inlining this should also work at the cost of additional `CALL` + `RET`.

**Comment by roman-khimov** [comment]
I think we have the main thing covered now after #1702 and inlinging/optimizations can be postponed to 0.94.0.
